README.md

```diff
@@ -15,4 +54,14 @@ $ pip install -r requirements.txt
 If something goes wrong, you can always:
 ```bash
 $ rm -rf $venv_path
-```
\ No newline at end of file
+```
+
+## Roundtrip tests
+The roundtrip tests hit the Neptune backend directly, which requires a valid Neptune configuration. As amundsen-gremlin CI does not currently have AWS configured, these tests do not run by default.
+
+In order to run the roundtrip tests:
+```bash
+$ python -m pytest --roundtrip .
+```
+=======
+```
```

Comment:
What's going on here? It renders a little weird in the markdown.

If you want a <hr/> I think `---` on its own line will do the trick (no backticks).

Reply:
Ah, this was a merge goof 

- [ ] fix